docs/test/index.js

```diff
@@ -83,7 +83,7 @@ t.test('html', async t => {
   // but we test for coverage
   t.test('files can link to root pages', async t => {
     await testBuildDocs(t, {
-      content: { 'test.md': '[link](/test)' },
+      content: { 'test.md': '[Test](/test)' },
```

Comment:
I understand this is a test as opposed to something that's read by a real user/screen reader, but people have a tendency to copy and paste from tests -- treating them as sample code, so unless there's a good reason not to fix a test, I prefer to offer a fix for it too.

Reply:
No worries, you're 100% right fixing even these.